src/focus.js

```diff
@@ -0,0 +1,37 @@
+﻿import {customAttribute} from 'aurelia-templating';
+import {inject} from 'aurelia-dependency-injection';
+
+var afterAttachedSetFocusDelay = 50;
+
+@customAttribute('focus')
+@inject(Element)
+export class Focus {
+    constructor(element) {
+        this.element = element;
+
+        this.focusListener = (e => this.value = true).bind(this);
+        this.blurListener = (e => this.value = false).bind(this);
```

Comment:
Since DOM libraries like jQuery use a `false` return value to stop event propagation, I think it would clarify intent to avoid returning the assigned value.